GalilSup/Db/galil_sans2d_front_det_limit_footer.gmc

```diff
@@ -1,4 +1,12 @@
 NO*************FOOTER*********************
+IF (fastStopFB=1)
+    oldecelC=_DCC;DCC=67107840;DCC=oldecelC;STC;
```

Comment:
If you're changing acceleration back before doing the stop it's not doing anything presumably? Where does the number 67107840 come from? Questions also apply to limit switch code below.

Reply:
`67107840` was the number used for the fast deceleration code under SECI in `LabVIEW Modules\Instruments\SANS2D\Sans2d Galil`. I don't know what it corresponds to physically (probably max acceleration that galil allows?) but I decided to make the behaviour of a collision avoidance stop consistent bump strips